cg/constants/devices.py

```diff
@@ -8,3 +8,9 @@ class DeviceType(Enum):
     BIONANO = auto()
     OXFORD_NANOPORE = auto()
     PACBIO = auto()
+
+
+class ParentClassId:
```

Comment:
```suggestion
class ParentClassFK:
```
or ForeignKey

Reply:
Decided to ignore the warning and use the foreign key strings instead